ara/api/migrations/0009_distincthost.py

```diff
@@ -1,26 +1,8 @@
-# Generated by Django 2.2.19 on 2021-04-03 17:38
+# Generated by Django 2.2.19 on 2021-07-07 19:57
 
 from django.db import migrations, models
 import django.db.models.deletion
 
-def update_latest_hosts(apps, schema_editor):
-    """ Computes the latest host for each host name to update the DistinctHost table """
-    # We can't import the model directly as it may be a newer
-    # version than this migration expects. We use the historical version.
-    db_alias = schema_editor.connection.alias
-    host_model = apps.get_model('api', 'Host')
-    distincthost_model = apps.get_model('api', 'DistinctHost')
-
-    updated = []
-    for host in host_model.objects.all():
-        if host.name in updated:
-            continue
-
-        # TODO: Should we sort by another column ?
-        latest_host = host_model.objects.filter(name=host.name).order_by('-updated')[0]
-        distincthost_model.objects.using(db_alias).create(name=host.name, latest_host=latest_host)
-        updated.append(host.name)
-
```

Comment:
Oops, did you mean to remove that ? We need it.

Reply:
sorry, my mistake. Now its there again